README.md

```diff
@@ -108,6 +108,47 @@ with the [contributors guide](https://github.com/pytorch/xla/blob/master/CONTRIB
 
 ## Getting Started
 
+Following here are guides for a simple single process, and multi process application.
+Multi processing is more complex, and is not compatible with SPMD.
+This tutorial does not dive into SPMD. For more on that, check our
+[SPMD guide](https://github.com/pytorch/xla/blob/master/docs/source/perf/spmd_basic.md).
+
+### Simple single process
+
+To update your exisitng training loop, make the following changes:
+
+```diff
++import torch_xla as xla
```

Comment:
nit: I've never seen anyone do `import torch_xla as xla`. I think keeping `import torch_xla` as-is will better align with the existing style.